### ISSUE TITLE
Charging Pods are Secretly Assisted-Teleportation Machines

### ISSUE THREAD

**Issue by Dumbdumn5** [ParadiseSS13/Paradise#13102]
**Issue Description**:
You, when grabbing somebody as an IPC, can store yourself into a charging station and, from any distance except for at Central Command, can translocate yourself instantly to its location from inside mechs, restraints, or otherwise, with the assistance of one other person pressing the 'eject' button.

**What did you expect to happen**: 
Normally charging pods aren't telepads.

**What happened instead**:
The charging pod was, instead, a telepad.

**Why is this bad/What are the consequences**:
You can cross z-level translocate yourself across the infinitesimal warp with speeds high enough to instantaneously eviscerate your fragile aluminum body, but appear fine. This can be done to escape a myriad of situations, but it's mostly good fun if you're bored and want to show off some magic tricks.

Basically, it's teleportation.

**Steps to reproduce the problem**:
Be IPC
Grab mob
Shimmy into charging pod
Your Mob gets stored into the pod but never actually enters it
Other person ejects charging pod
Teleport on top of now ejected pod
Profit

**When did the problem start happening**:
Not Applicable

**Extra information**:
It's one hell of a fun gag.

**Comment by Contrabang** [comment]
Resolved by #24709 and https://github.com/ParadiseSS13/Paradise/pull/24246
